spec/e2e/ivc.spec.ts

```diff
@@ -27,4 +29,91 @@ describe("IVC", () => {
 
     expect(scores as MSSIMValues).toEqual(results);
   });
+
+  it("should match stored mssims (weber)", async () => {
+    const expected = await getJSONScores(weberScores, path, "bmp");
+    const results = Object.entries(expected)
+      .map(([key, { file, reference }]): [string, number] => {
+        if (key === "isabe_jpeg_r1.bmp") {
+          debugger;
+        }
+        const { mssim, ssim_map } = ssim(reference, file, { ssim: "weber" });
+
+        return [key, roundTo(mssim, 3)];
+      })
+      .reduce(
+        (acc, [key, result]) => ({
+          ...acc,
+          [key]: result
+        }),
+        {} as MSSIMValues
+      );
+    //
+    const referenceScores = scores as MSSIMValues;
+    const newV: any = {};
+    let newMean = 0;
+    let newS = 0;
+    for (let score in referenceScores) {
+      const refVal = referenceScores[score];
+      const weberNewVal = results[score];
+      const distWeberNew = Math.abs(refVal - weberNewVal);
+      newV[score] = {
+        mssim: weberNewVal,
+        distance: distWeberNew,
+        ref: refVal
+      };
+      const newMeanR = newMean + (distWeberNew - newMean) / Object.keys(newV).length;
+      newS = newS + (distWeberNew - newMean) * (distWeberNew - newMeanR);
+      newMean = newMeanR;
+    }
+
+    // console.log(JSON.stringify(results));
+
+    const newVar = newS / (Object.keys(newV).length - 1);
+    // console.log(`mean: ${newMean} variance: ${newVar}`);
+    expect(roundTo(newMean,4 )).toEqual(0.0202);
+    expect(roundTo(newVar, 6)).toEqual(0.000211);
+    expect(results).toEqual(weberScores as MSSIMValues);
+  }, 70000);
```

Comment:
we should be able to [change the default timeout as part of the jest config](https://jestjs.io/docs/en/configuration#testtimeout-number) instead

Reply:
This is a style thing, really.  It makes sense to do it on your long-running tests individually that way if another short test breaks it doesn't get the benefit of a long time out.  We should probably add some default constants though for short and long running tests where it's appropriate.  Generally speaking, I don't have any objection one way or the other though.

Reply:
Yeah and I actually think I agree with you, this shouldn't be the default timeout, this makes sense for the slow tests only as you have it now 👍